GUI.py

```diff
@@ -194,7 +196,7 @@ def transcribe(self) :
         normalized_audio = normalize(pre_normalized_audio)
         # transcribed audio is now using normalized audiofile
         self.convertToWAV(normalized_audio)
-        transcribedAudio = diarizationAndTranscription.diarizeAndTranscribe("converted.wav")
+        transcribedAudio = diarizationAndTranscription.diarizeAndTranscribe("SpeechTranscription\converted.wav")
```

Comment:
May have to change once the .exe is built and program exists in another filepath.